executor/adapter.go

```diff
@@ -332,17 +332,25 @@ func (a *ExecStmt) RebuildPlan(ctx context.Context) (int64, error) {
 	return a.InfoSchema.SchemaMetaVersion(), nil
 }
 
-func (a *ExecStmt) setPlanLabelForTopSQL(ctx context.Context) context.Context {
-	if !topsqlstate.TopSQLEnabled() {
-		return ctx
+func isFastPlan(p plannercore.Plan) bool {
```

Comment:
Is it possible to have some unit tests to ensure that isFastPlan is indeed working for these type of SQLs you give in the comment?